app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss

```diff
@@ -1,81 +1,81 @@
-.govuk-previous-and-next-navigation {
+.app-c-pagination {
   display: block;
   margin-top: $gutter;
   margin-bottom: $gutter;
   margin-left: -$gutter-half;
   margin-right: -$gutter-half;
+}
 
-  ul {
-    margin: 0;
-    padding: 0;
-  }
-
-  li {
-    @include core-16($line-height: (20 / 16));
-    float: left;
-    list-style: none;
-    text-align: right;
-    margin: 0;
-    padding: 0;
-    width: 50%;
+.app-c-pagination__list {
+  margin: 0;
+  padding: 0;
 
-    a {
-      display: block;
-      padding: $gutter-half;
-      text-decoration: none;
+  &:after {
+    content: "";
+    display: block;
+    clear: both;
+  }
+}
 
-      &:visited {
-        color: $link-colour;
-      }
+.app-c-pagination__item {
+  @include core-16($line-height: (20 / 16));
+  float: left;
+  list-style: none;
+  text-align: right;
+  margin: 0;
+  padding: 0;
+  width: 50%;
+}
 
-      &:hover,
-      &:active {
-        background-color: $canvas-colour;
-      }
+.app-c-pagination__link {
+  display: block;
+  padding: $gutter-half;
+  text-decoration: none;
 
-      .pagination-part-title {
-        @include core-27($line-height: (33.75 / 27));
-        display: block;
-      }
-    }
+  &:visited {
+    color: $link-colour;
   }
 
-  .previous-page {
-    float: left;
-    text-align: left;
+  &:hover,
+  &:active {
+    background-color: $canvas-colour;
   }
+}
 
-  .next-page {
-    float: right;
-    text-align: right;
-  }
+.app-c-pagination__link-title {
+  @include core-27($line-height: (33.75 / 27));
+  display: block;
+}
 
+.app-c-pagination__item--previous {
   @include media-down(mobile) {
-    .previous-page,
-    .next-page {
-      float: none;
-      width: 100%;
-    }
-
-    .next-page a {
-      text-align: right;
-    }
+    float: none;
+    width: 100%;
   }
 
-  .pagination-icon {
-    display: inline-block;
-    margin-bottom: 1px;
-    height: .482em;
-    width: .63em;
+  float: left;
+  text-align: left;
+}
+
+.app-c-pagination__item--next {
+  @include media-down(mobile) {
+    float: none;
+    width: 100%;
   }
 
-  .pagination-label {
-    display: inline-block;
-    margin-top: 0.1em;
-    text-decoration: underline;
+  float: right;
+  text-align: right;
+}
 
-    &:empty {
```

Comment:
I wrote this but didn't properly document why (I'm a bad person), I think this might be a workaround for an issue in one of the applications.

My gut was that it was something to do with manuals: https://www.gov.uk/hmrc-internal-manuals/admin-law-manual/adml4000 since they don't have labels, but you're right they dont render at all.

Let's just be extra careful when we deploy to check all the places this is used.

https://govuk-publishing-frontend-search.glitch.me/?q=govuk_component/previous_and_next_navigation